modules/objdetect/test/test_arucodetection.cpp

```diff
@@ -638,6 +638,94 @@ TEST(CV_ArucoDetectMarkers, regression_contour_24220)
     }
 }
 
+TEST(CV_ArucoMultiDict, addRemoveDictionary)
+{
+    aruco::ArucoDetector detector;
+    detector.addDictionary(aruco::getPredefinedDictionary(aruco::DICT_5X5_100));
+    const auto& dicts = detector.getDictionaries();
+    ASSERT_EQ(dicts.size(), 2ul);
```

Comment:
Please add comment to the detector constructor that it uses one of predefined dictionaries by default. Value `2` is not obvious here.